### PR TITLE
i18n: use consistent color values (RGB/HEX/HSL)

### PR DESCRIPTION
**Pull request by ramiy**
## Description

The color-picker uses full color formats in some case and the abbreviations is others. This PR replaces all the i18n strings to use consistent color values ( **RGB** / **HEX** / **HSL** ).

## How has this been tested?

Using the https://translate.wordpress.org/ platform

## Screenshots

i18n-colors-1

i18n-colors-2

## Types of changes

String changes inside `__()` functions.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/components/src/color-picker/inputs.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ export class Inputs extends Component {
 			return (
 				<div className="components-color-picker__inputs-fields">
 					<Input
-						label={ __( 'Color value in hexadecimal' ) }
+						label={ __( 'Color value in HEX' ) }
```
Comment (swissspidy) [inline packages/components/src/color-picker/inputs.js]: Hex isn't an acronym like HSL or RGB and should not be capitalized.

File: packages/components/src/color-picker/inputs.js [hunk 2]
```diff
@@ -103,12 +103,12 @@ export class Inputs extends Component {
...
 			this.setState( { view: 'hsl' } );
 
-			speak( __( 'Hue/saturation/lightness mode active' ) );
+			speak( __( 'HSL mode active' ) );
 		} else if ( this.state.view === 'hsl' ) {
 			if ( this.props.hsl.a === 1 ) {
 				this.setState( { view: 'hex' } );
 
-				speak( __( 'Hex color mode active' ) );
+				speak( __( 'HEX color mode active' ) );
```
Comment (swissspidy) [inline packages/components/src/color-picker/inputs.js]: Hex isn't an acronym like HSL or RGB and should not be capitalized.

Comment (ramiy) [reply packages/components/src/color-picker/inputs.js]: Fixed.

File: packages/components/src/color-picker/inputs.js (hunk 1 above)
Comment (ramiy) [reply packages/components/src/color-picker/inputs.js]: Fixed.

**Comment by ramiy** [comment]
@swissspidy Can you please review the changes?

File: packages/components/src/color-picker/inputs.js [hunk 3]
```diff
@@ -103,7 +103,7 @@ export class Inputs extends Component {
 		} else if ( this.state.view === 'rgb' ) {
 			this.setState( { view: 'hsl' } );
 
-			speak( __( 'Hue/saturation/lightness mode active' ) );
+			speak( __( 'HSL mode active' ) );
```
Comment (swissspidy) [inline packages/components/src/color-picker/inputs.js]: Given that text is spoken aloud to users with screen readers, `Hue/saturation/lightness` might be way better to understand for them than `HSL`.

File: packages/components/src/color-picker/inputs.js [hunk 4]
```diff
@@ -162,7 +162,7 @@ export class Inputs extends Component {
 			return (
 				<div className="components-color-picker__inputs-fields">
 					<Input
-						label={ __( 'Color value in hexadecimal' ) }
+						label={ __( 'Color value in Hex' ) }
```
Comment (swissspidy) [inline packages/components/src/color-picker/inputs.js]: Isn't `hexadecimal` clearer to understand and easier to translate than `Hex` (which should be lowercase btw)?

**Comment by swissspidy** [comment]
I'm not sure if these changes are really worth it as I think they make things harder to translate.

**Comment by michelleweber** [comment]
Is this copy you need an editor to weigh in on, or is it more an internationalization/accessibility question of what terms to use?

**Comment by swissspidy** [comment]
The latter :-)

Feel free to remove the label in case I wrongly assigned it 😕

**Comment by michelleweber** [comment]
No worries! Just wanted to make sure you're getting what you need :)

File: packages/components/src/color-picker/inputs.js (hunk 3 above)
Comment (ramiy) [reply packages/components/src/color-picker/inputs.js]: `HSL` is a not a new word, we already use it in other strings. This PR is just for consistency - to use the same work in other places.

As for screen readers - just as regular users understand the meaning from the context, so are screen reader users.

File: packages/components/src/color-picker/inputs.js (hunk 4 above)
Comment (ramiy) [reply packages/components/src/color-picker/inputs.js]: Industry standard is `Hex`, or in some case `HEX`. The full `hexadecimal` is usually used in documentation.

Comment (ramiy) [reply packages/components/src/color-picker/inputs.js]: Besides, We are already using `Hex` in other string, this PR is for creating consistency in color values.

**Comment by afercia** [comment]
@gziolo this PR partially overlaps with #11303.

I have no strong opinions about the label `Color value in hexadecimal`. However, the `speak()` messages are meant to make clear to screen reader users what happened. I wouldn't use abbreviations as hearing "RGB mode active" is less clear than "Red Green Blue mode active". Also, I'm not sure why it's a "mode" 🙂 This is a color value format: the toggle button says "Change color **format**".

I'd change all the 4 speak messages to use an extended wording:

Red Green Blue format active
Hue Saturation Lightness format active
Hexadecimal format active

**Comment by gziolo** [comment]
Let's consolidate work in #11303 as it looks like it has a much wider set of changes applied and goes in the opposite direction. Thanks @ramiy for opening this PR and your contribution. Let's make sure all labels are clarified in #11303.

**Pull request closed without merging**